[Bug] Crash: Support AnkiDroid dialog - Don't keep activities

###### Reproduction Steps

1. Have Don't keep activities enabled
2. Open Support AnkiDroid dialog (on the navigation drawer)
3. Minimize the app
4. Open it again


###### Expected Result

No crashes

###### Actual Result

Crash

###### Debug info
```
AnkiDroid Version = 2.16alpha60

Android Version = 12

Manufacturer = samsung

Model = SM-N770F

Hardware = exynos9810

Webview User Agent = Mozilla/5.0 (Linux; Android 12; SM-N770F Build/SP1A.210812.016; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.4896.88 Mobile Safari/537.36

ACRA UUID = 037bbc86-38a4-4ad7-9c42-05870e9b6086

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/037bbc86-38a4-4ad7-9c42-05870e9b6086

```
android.os.BadParcelableException: Parcelable protocol requires a Parcelable.Creator object called CREATOR on class com.ichi2.anki.dialogs.HelpDialog$LinkItem
at android.os.Parcel.readParcelableCreator(Parcel.java:3431)
at android.os.Parcel.readParcelable(Parcel.java:3336)
at android.os.Parcel.readValue(Parcel.java:3238)
at android.os.Parcel.readListInternal(Parcel.java:3698)
at android.os.Parcel.readArrayList(Parcel.java:2822)
at android.os.Parcel.readValue(Parcel.java:3259)
at android.os.Parcel.readArrayMapInternal(Parcel.java:3635)
at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:292)
at android.os.BaseBundle.unparcel(BaseBundle.java:236)
at android.os.Bundle.getParcelableArrayList(Bundle.java:1056)
at com.ichi2.anki.dialogs.RecursivePictureMenu.onCreateDialog(RecursivePictureMenu.kt:1)
at androidx.fragment.app.DialogFragment.prepareDialog(DialogFragment.java:4)
at androidx.fragment.app.DialogFragment.onGetLayoutInflater(DialogFragment.java:3)
at androidx.fragment.app.Fragment.performGetLayoutInflater(Fragment.java:1)
at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:4)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:20)
at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:3)
at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:5)
at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3)
at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:4)
at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:1)
at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:6)
at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:1)
at com.ichi2.anki.AnkiActivity.onStart(AnkiActivity.java:2)
at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1455)
at android.app.Activity.performStart(Activity.java:8307)
at android.app.ActivityThread.handleStartActivity(ActivityThread.java:4056)
at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2434)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:226)
at android.os.Looper.loop(Looper.java:313)
at android.app.ActivityThread.main(ActivityThread.java:8633)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1133)
Caused by: java.lang.NoSuchFieldException: CREATOR
at java.lang.Class.getField(Class.java:1635)
at android.os.Parcel.readParcelableCreator(Parcel.java:3408)
... 38 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt
@@ -147,7 +147,7 @@ object HelpDialog {
         private constructor(`in`: Parcel?) : super(`in`!!)
 
         companion object {
-            val CREATOR: Parcelable.Creator<RateAppItem?> = object : Parcelable.Creator<RateAppItem?> {
+            @JvmField val CREATOR: Parcelable.Creator<RateAppItem?> = object : Parcelable.Creator<RateAppItem?> {
                 override fun createFromParcel(`in`: Parcel): RateAppItem {
                     return RateAppItem(`in`)
                 }
@@ -189,7 +189,7 @@ object HelpDialog {
         }
 
         companion object {
-            val CREATOR: Parcelable.Creator<LinkItem?> = object : Parcelable.Creator<LinkItem?> {
+            @JvmField val CREATOR: Parcelable.Creator<LinkItem?> = object : Parcelable.Creator<LinkItem?> {
                 override fun createFromParcel(`in`: Parcel): LinkItem {
                     return LinkItem(`in`)
                 }
@@ -231,7 +231,7 @@ object HelpDialog {
         }
 
         companion object {
-            val CREATOR: Parcelable.Creator<FunctionItem?> = object : Parcelable.Creator<FunctionItem?> {
+            @JvmField val CREATOR: Parcelable.Creator<FunctionItem?> = object : Parcelable.Creator<FunctionItem?> {
                 override fun createFromParcel(`in`: Parcel): FunctionItem {
                     return FunctionItem(`in`)
                 }

--- a/AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt
@@ -35,6 +35,7 @@ import com.ichi2.anki.exception.UserSubmittedException
 import com.ichi2.utils.AdaptionUtil.isUserATestClient
 import com.ichi2.utils.IntentUtil.canOpenIntent
 import com.ichi2.utils.IntentUtil.tryOpenIntent
+import com.ichi2.utils.KotlinCleanup
 import org.acra.ACRA
 import org.acra.config.DialogConfigurationBuilder
 import org.acra.config.LimiterData
@@ -133,6 +134,7 @@ object HelpDialog {
         return createInstance(itemList, R.string.help_title_support_ankidroid)
     }
 
+    @KotlinCleanup("Convert to @Parcelize")
     class RateAppItem : RecursivePictureMenu.Item, Parcelable {
         constructor(@StringRes titleRes: Int, @DrawableRes iconRes: Int, analyticsRes: String?) : super(titleRes, iconRes, analyticsRes)
 
@@ -159,6 +161,7 @@ object HelpDialog {
         }
     }
 
+    @KotlinCleanup("Convert to @Parcelize")
     class LinkItem : RecursivePictureMenu.Item, Parcelable {
         @StringRes
         private val mUrlLocationRes: Int
@@ -201,6 +204,7 @@ object HelpDialog {
         }
     }
 
+    @KotlinCleanup("Convert to @Parcelize")
     class FunctionItem : RecursivePictureMenu.Item, Parcelable {
 
         private val mFunc: ActivityConsumer

